Userland/Applications/KeyboardMapper/KeyButton.cpp

```diff
@@ -64,17 +64,22 @@ void KeyButton::mousemove_event(GUI::MouseEvent& event)
 
     Gfx::IntRect key_cap_face_rect = { rect().x() + 7, rect().y() + 4, rect().width() - 14, rect().height() - 14 };
 
-    if (key_cap_face_rect.contains(event.position())) {
-        window()->set_cursor(Gfx::StandardCursor::Hand);
-        return;
-    }
-    window()->set_cursor(Gfx::StandardCursor::Arrow);
+    set_face_hovered(key_cap_face_rect.contains(event.position()));
 
     AbstractButton::mousemove_event(event);
 }
 
 void KeyButton::leave_event(Core::Event& event)
 {
-    window()->set_cursor(Gfx::StandardCursor::Arrow);
+    set_face_hovered(false);
     AbstractButton::leave_event(event);
 }
+
+void KeyButton::set_face_hovered(bool value)
+{
+    m_face_hovered = value;
+    if (m_face_hovered)
+        window()->set_cursor(Gfx::StandardCursor::Hand);
+    else
+        window()->set_cursor(Gfx::StandardCursor::Arrow);
```

Comment:
Instead of changing the cursor of the entire window, can we use `GUI::Widget::set_override_cursor()` instead?

Reply:
Yes, that indeed seems to be a possibillity!